db/config.cc

```diff
@@ -1317,7 +1317,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , view_building(this, "view_building", value_status::Used, true, "Enable view building; should only be set to false when the node is experience issues due to view building.")
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Unused, true, "Enable SSTables 'mc' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
-    , sstable_format(this, "sstable_format", liveness::LiveUpdate, value_status::Used, "me", "Default sstable file format", {"md", "me", "ms"})
+    , sstable_format(this, "sstable_format", liveness::LiveUpdate, value_status::Used, "ms", "Default sstable file format", {"md", "me", "ms"})
```

Comment:
You convinced me to be more conservative, so let's enable this for new clusters only by naming `ms` in the default scylla.yaml.

Reply:
For the sake of being conservative, should we enable Trie indexes per table, so they could be used only when beneficial?

Reply:
It will end in an endless mix of configurations. No one has the bandwidth to evaluate different index types per table.

I don't believe it's worthwhile. I think the penalty from Trie when it is a regression is small, while the benefit when it is not is very large.